solo/methods/barlow_twins.py

```diff
@@ -17,32 +17,35 @@
 # OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-import argparse
 from typing import Any, List, Sequence
 
+import omegaconf
 import torch
 import torch.nn as nn
 from solo.losses.barlow import barlow_loss_func
 from solo.methods.base import BaseMethod
+from solo.utils.misc import omegaconf_select
 
 
 class BarlowTwins(BaseMethod):
-    def __init__(
-        self, proj_hidden_dim: int, proj_output_dim: int, lamb: float, scale_loss: float, **kwargs
-    ):
+    def __init__(self, cfg: omegaconf.DictConfig):
```

Comment:
it was nice tho to see the arguments and their type in the init

Reply:
We can have the types in the init, like self.proj_hidden_dim : int = cfg.method_kwargs.proj_hidden_dim